### ISSUE TITLE
Amazon Payments can fail if edd_checkout_global scripts are in footer

### ISSUE THREAD

**Issue by cklosowski** [easydigitaldownloads/easy-digital-downloads#5996]
We moved the `edd-checkout-global` JS to the footer in #4860, however this causes an issue with the Amazon Payments gateway. If something triggers needing the EDD global vars earlier, than it will load in the header, but on a stock install, it loads in the footer causing a JS error mentioning that the `taxes_enabled` property of undefined.

We need to move it to the header of Amazon Payments is loaded.

**Comment by cklosowski** [comment]
To replicate, setup a base EDD installation with Amazon enabled.

When you get to checkout and authorize Amazon, you'll get an error stating that the `taxes_enabled` can't be access on `undefined`. If you view source you'll see that our `edd-checkout-global` JS file and all it's localized scripts will be loaded in the footer.

With this PR the error goes away, and the scripts are all loaded in the header to allow them to be accessed by on-page JS.

**Comment by SDavisMedia** [comment]
Went through this on a call with @cklosowski. Replicated the behavior on `master` and confirmed fixed on `issue/5996`. :+1:
